.github/workflows/main.yaml

```
@@ -17,31 +18,81 @@
+      - name: Setup Android emulator
+        uses: ReactiveCircus/android-emulator-runner@v2
+        with:
+          api-level: 29
+          arch: x86_64
+          profile: Nexus 6
+          script: |
+            flutter doctor -v
+            flutter test integration_test/app_test.dart --coverage
+
+      - name: Process integration test coverage
+        run: |
```

actionlint (1.7.4)

shellcheck reported issue in this script: SC2001:style:6:19: See if you can use ${variable//search/replace} instead

(shellcheck)